gateway/src/main.rs

```diff
@@ -141,7 +140,7 @@ async fn start(db: SqlitePool, fs: PathBuf, args: StartArgs) -> io::Result<()> {
     let acme_client = AcmeClient::new();
 
     let mut api_builder = ApiBuilder::new()
-        .with_service(Arc::clone(&gateway))
+        .with_service(Arc::<GatewayService>::clone(&gateway))
```

Comment:
I see you did this in a couple of places, I'm curios why this is needed?

Reply:
I've developed for some time with 1.67.1 before I've changed to 1.65. At that time my vs code recommended me to include the turbofish operator or I couldn't compile the source code (can't remember which one). I blamed it on the 1.67.1 version at that time but I can not reproduce it anymore. I'll remove the turbofish operator because it compiles just fine without it.